src/gwt/panmirror/src/editor/src/api/widgets/navigation-tree.tsx

```diff
@@ -56,7 +56,6 @@ const kNavigationTreeIndent = 8;
 // like you would use to navigate a hierarchical file system.
 export const NavigationTree: React.FC<NavigationTreeProps> = props => {
   const style: CSSProperties = {
-    overflowY: 'scroll',
```

Comment:
Is the idea that this will never overflow? Or is the overflow handled elsewhere?

Reply:
It is handled elsewhere (the tree itself uses overflow auto, so there is literally no reason for this). I can't explain why it made its way in other than perhaps I left it in after doing some testing of scroll behavior...